src/commands/Games/higherLower.ts

```diff
@@ -0,0 +1,252 @@
+import { SkyraCommand } from '../../lib/structures/SkyraCommand';
+import { CommandStore, KlasaMessage } from 'klasa';
+import { MessageEmbed } from 'discord.js';
+import { LongLivingReactionCollector, LLRCData } from '../../lib/util/LongLivingReactionCollector';
+import { UserSettings } from '../../lib/types/settings/UserSettings';
+import { Time } from '../../lib/util/constants';
+import { getColor } from '../../lib/util/util';
+import { Slotmachine } from '../../lib/util/Games/Slotmachine';
+
+enum ReactionEmoji {
+	HIGHER = '⬆',
+	LOWER = '⬇',
+	CANCEL = '<:redCross:637706251257511973>',
+	OK = '<:greenTick:637706251253317669>',
+	CASHOUT = '💰'
+}
+
+const enum EndingAction {
+	PLAY,
+	STOP,
+	TIMEOUT
+}
+
+export default class extends SkyraCommand {
+
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			aliases: ['hilo', 'higherlower', 'hl'],
+			bucket: 2,
+			cooldown: 7,
+			description: language => language.tget('COMMAND_HIGHERLOWER_DESCRIPTION'),
+			extendedHelp: language => language.tget('COMMAND_HIGHERLOWER_EXTENDED'),
+			requiredPermissions: ['ADD_REACTIONS', 'MANAGE_MESSAGES', 'EMBED_LINKS'],
+			runIn: ['text'],
+			usage: '<50|100|200|500|1000|2000|5000|10000>'
+		});
+	}
+
+	public async run(message: KlasaMessage, [text]: [string]) {
+		const wager = Number(text);
+		const balance = message.author.settings.get(UserSettings.Money);
+		if (balance < wager) throw message.language.tget('GAMES_NOT_ENOUGH_MONEY', balance);
+
+		let emojiPromiseContext: EmojiPromiseContext = { resolve: undefined, timeout: undefined };
+
+		const game: HigherLowerGameData = Object.seal({
+			llrc: new LongLivingReactionCollector(this.client, async reaction => {
+				// Ignore if resolve is not ready
+				if (typeof emojiPromiseContext.resolve !== 'function'
+					// Run the collector inhibitor
+					|| await this.reactionInhibitor(message, game.gameMessage!, reaction)) return;
+				this.client.clearTimeout(emojiPromiseContext.timeout!);
+				emojiPromiseContext.resolve(reaction.emoji.name as ReactionEmoji);
+			}),
```

Comment:
The LLRC's collector is meant to be a method, not a function.

Reply:
Uhh... wdym

Reply:
It should be something like `new LongLivingReactionCollector(this.client, this.someMethodWhenThingsAreCollected.bind(this))`